README.md

```diff
@@ -8,27 +8,57 @@ products:
 
 ## Prerequisites
 
 1. [npm](https://www.npmjs.com/get-npm)
-2. [Node.js](https://nodejs.org/en/download/)
+2. [Node.js (v14)](https://nodejs.org/en/download/)
+3.  Create an Azure account with an active subscription. For details, see [Create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+4. Create an Azure Communication Services resource. For details, see [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource). You'll need to record your resource **connection string** for this quickstart.
 
 ## Code structure
 
-* ./src: client side source code
-* ./webpack.config.js: Project bundler. Has a simple local server for user token provisioning.
-* ./config.json: configuration file for specifying the connectiong string.
+* [`./Project/src`](./Project/src): Client side source code
+* [`./Project/src/app/App.js`](./Project/src/app/App.js): Entry point into the client source code 
+* [`./Project/webpack.config.js`](./Project/webpack.config.js): Project bundler. Has a simple local server for user token provisioning.
+* [`./Project/config.json`](./Project/config.json): configuration file for specifying the connectiong string.
 
 ## Before running the sample for the first time
 1. git clone https://github.com/Azure-Samples/communication-services-web-calling-tutorial
 2. cd communication-services-web-calling-tutorial/Project
 3. Get a connection string by provisioning an Azure Communication Services resource from the Azure portal. Use the connection string as value for key `connectionString` in config.json file.
-4. npm install
-5. npm run build
-6. npm run start
-7. Open localhost:5000 in a browser. (Supported browsers are Chrome, Edge Chromium, and Safari)
+
+## Local Run
+1. Open a command prompt in the `Project` folder
+2. `npm install`
+3. `npm run start`
+4. Open localhost:5000 in a browser. (Supported browsers are Chrome, Edge Chromium, and Safari)
+
+## Deployment to Azure from VS Code
+1. Open a command prompt in the `Project` folder
+2. `npm install`
+3. `npm run build`
+4. Download the [Azure Plugin](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureresourcegroups) and the [Azure Static Web App Plugin](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps) for VS Code.
+5. In the Azure plugin pane, click `+`, and click `Create App Service Web App`.
+6. Enter your subscription
+7. Enter a unique app name
+8. Select `Node 14` as the Runtime.
+
+Your App Service should be configured
+
+## Deployment to Azure
+
+Use the blue buttons above.
```

Comment:
So this will bring me to a portal where i have to do more things. You need to explain what i have to do in the portal.

Also, i should be able to deploy this manually therough the azure app service extension in vscode:
![image](https://user-images.githubusercontent.com/65419124/192645402-056eba03-ebc3-4336-84ef-09b9867e1667.png)

I right click on an app service and then click on "Deply to web app". We have always been deploying this way so this mehtod of deplyment should also work. It will deploy to <app service name>.azurewebsites.net